assets/blocks/brownCornerRoof.block

```diff
@@ -0,0 +1,8 @@
+{
+    "displayName": "Brown Corner Roof",
+    "family": "horizontal",
+    "shape": "engine:slopecorner",
```

Comment:
I don't think it is necessary to declare own block types for `brownCornerRoof` and `brownRoof` (we'll need the texture, though). 
In-game, you can retrieve the equivalent blocks via

```
give hjerlhede:brownroof 16 engine:slopecorner
give hjerlhede:brownroof 16 engine:slope
```

Reply:
Both giving brownRoof somehow.